capstone/src/main/java/com/google/sps/servlets/DashboardHandlerServlet.java

```diff
@@ -15,6 +24,20 @@
 @WebServlet("/dashboard-handler")
 public class DashboardHandlerServlet extends HttpServlet {
 
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        PreparedQuery results = datastore.prepare(new Query(Classroom.CLASSROOM_ENTITY_NAME));
+    
+        List<Classroom> classrooms = new ArrayList<>();
```

Comment:
There is no need to upcast this. In this case it doesnt make a difference so there is no need to change it if you dont want, but as a rule, I wouldnt upcast unless there is a reason to

Reply:
Got it.